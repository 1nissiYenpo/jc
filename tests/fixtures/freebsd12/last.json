[{"user": "kbrazil", "tty": "pts/0", "hostname": "192.168.71.1", "login": "Thu May 28 21:29", "logout": "still logged in"}, {"user": "kbrazil", "tty": "pts/0", "hostname": "192.168.71.1", "login": "Thu May 28 04:13", "logout": "21:29", "duration": "17:16"}, {"user": "kbrazil", "tty": "pts/0", "hostname": "192.168.71.1", "login": "Wed May 27 16:09", "logout": "04:13", "duration": "12:03"}, {"user": "root", "tty": "ttyv0", "hostname": null, "login": "Wed May 27 12:48", "logout": "still logged in"}, {"user": "kbrazil", "tty": "pts/0", "hostname": "192.168.71.1", "login": "Wed May 27 12:47", "logout": "16:09", "duration": "03:22"}, {"user": "kbrazil", "tty": "ttyv0", "hostname": null, "login": "Wed May 27 12:46", "logout": "12:48", "duration": "00:01"}, {"user": "boot time", "tty": null, "hostname": null, "login": "Wed May 27 12:46"}]
