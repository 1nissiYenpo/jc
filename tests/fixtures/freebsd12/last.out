kbrazil    pts/0    192.168.71.1           Thu May 28 21:29   still logged in
kbrazil    pts/0    192.168.71.1           Thu May 28 04:13 - 21:29  (17:16)
kbrazil    pts/0    192.168.71.1           Wed May 27 16:09 - 04:13  (12:03)
root       ttyv0                           Wed May 27 12:48   still logged in
kbrazil    pts/0    192.168.71.1           Wed May 27 12:47 - 16:09  (03:22)
kbrazil    ttyv0                           Wed May 27 12:46 - 12:48  (00:01)
boot time                                  Wed May 27 12:46

utx.log begins Wed May 27 12:46:05 PDT 2020
